modules/Microsoft.Compute/virtualMachineScaleSets/.test/.scripts/New-SSHKey.ps1

```diff
@@ -0,0 +1,19 @@
+﻿param(
+    [string] $SSHKeyName,
+    [string] $ResourceGroupName
+)
+
+if (-not ($sshKey = Get-AzSshKey -ResourceGroupName $ResourceGroupName | Where-Object { $_.Name -eq $SSHKeyName })) {
+    Write-Verbose "No SSH key [$SSHKeyName] found in Resource Group [$ResourceGroupName]. Generating new." -Verbose
+    $null = ssh-keygen -f generated -N (Get-Random -Maximum 99999)
+    $publicKey = Get-Content 'generated.pub' -Raw
+    # $privateKey = cat generated | Out-String
```

Comment:
Should this be removed?

Reply:
I'd suggest to keep it. It essentially shows you how you 'could' also retrieve the private key from what the `ssh-keygen` function produced. It just happends that we don't need it in our test - hence it is only a comment. However, I don't know about you, but I often re-use these kinds of scripts outside of their original context ;) Hence, I guess no harm in keeping it,

Reply:
Fair enough :)